Tools/Scripts/libraries/resultsdbpy/resultsdbpy/view/static/js/expectations.js

```diff
@@ -62,7 +63,7 @@ class Expectations
             if (i > -1)
                 r.splice(i, 1);
             if (expectation === 'FAIL')
-                ['TEXT', 'AUDIO', 'IMAGE'].forEach(expectation => {
+                ['TEXT', 'AUDIO'].forEach(expectation => {
```

Comment:
I don't think we want to take `IMAGE` out here. What this bit of code does is let a test marked as expected to `FAIL` show a green check if that test actually has an `IMAGE` failure, which I think should still be true after your change, right?

Reply:
I think so. Let me add it back.